sr_ur_arm_calibration_loader/src/sr_ur_arm_calibration_loader/sr_ur_arm_calibration_loader.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/python
+
+# Copyright 2019 Shadow Robot Company Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import rospy
+import roslaunch
+import rospkg
+import Tkinter as tk
+import tkMessageBox as messageBox
+import paramiko
+import sys
+import yaml
+from rosparam import upload_params
+
+
+class SrUrLoadCalibrationExceptions(Exception):
+    pass
+
+
+class NoArmsSpecified(SrUrLoadCalibrationExceptions):
+    pass
+
+
+class ArmTypeNotRecognised(SrUrLoadCalibrationExceptions):
+    pass
+
+
+class DifferentArmTypes(SrUrLoadCalibrationExceptions):
+    pass
+
+
+class SrUrLoadCalibration(object):
+    def __init__(self, arm_info_in=[]):
+        self._default_kinematics_config = ''
+        self._arm_calibrations_folder = ''
+        self._CONST_UR_ARM_SSH_USERNAME = "root"
+        self._CONST_UR_ARM_SSH_PASSWORD = "easybot"
```

Comment:
shouldn't this be a variable, not hardcoded

Reply:
it's defined as a CONST

Reply:
how do we deal with the fact if the password or username change in the future (and are we ok with them being in a public repo)? Shouldn't there be some easy way to changing a configuration file and reading these values from the configuration file?